udata/core/dataset/rdf.py

```diff
@@ -126,7 +126,23 @@ def owner_to_rdf(dataset, graph=None):
     return
 
 
-def ogc_service_to_rdf(dataset, resource, graph=None, is_hvd=False):
+def detect_ogc_service(resource):
```

Comment:
Is it worth adding types?

Reply:
Done in https://github.com/opendatateam/udata/pull/3241/commits/b7de4d998e93edecf402e5da35fabb3bf9acd1af

Also added on other `x_to_rdf` functions in this file since I was at it